docs/src/content/tutorial/create_api_doc.md

```diff
@@ -0,0 +1,242 @@
+---
+name: Creating API Documentation
+menu: Tutorial
+---
+
+# Creating an API Documentation using hydra
+
+> This tutorial assumes you are familiar with _hydra, the general workflow_ and have a basic understanding of _python programming language._ To get familiar with these technology, follow the workflow tutorial.
```

Comment:
I think you would probably be doing this already but just saying so we don't forget, we can hyperlink `workflow tutorial` to its correct URL.

Reply:
Yeah! The links currently are a little volatile. I will add a TODO comment and do it later afterward. Thanks!